### DISCUSSION TITLE
hilights from SMX file

### DISCUSSION THREAD

**Discussion by umeca74** [sumatrapdfreader/sumatrapdf#2958] [Q&A]
I was using an older version of sumatra PDF and saved highlights in a separate SMX sidecar file. Entries look like that

```
[@meta]
version = 3.2.11133

[highlight]
page = 180
rect = 353 449 187 26
color = #ffff60
opacity = 0.8

```

using the latest sumatra none of these highlights are used. It's as if sumatra isn't recognizing the SMX. Is there some setting I can use or the feature is no longer supported?

thanks
nikos

**Comment by GitHubRulesOK** [comment]
The support was removed (3.3?) when new pdf annotation was conflicting there were both only for a short while.
If you have the older 3.2.11133 version for creation keep it but AFAIK stable 3.2 was last to display them (not build them)
(or perhaps stick with XP 3.1.2 for display as when they were used as best) 

You can emulate new ones for 3.2 by say copy and paste but its slow by hand and beware the difference in scales
where pdf overlay units 1/72" are different to say image overlay units 1/96" so a factor of 4/3 (133%/75%)

**Reply by umeca74** [reply]
I found a solution, I used the old version to read the file then saved it as a new PDF, sorted!
